src/main/java/core/basesyntax/service/ReadDataFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReadDataFromFile {
+    List<String> readStatisticFromFile(String fromFileName);
+}
```

Comment:
ReaderService can read no only statistic
```suggestion
public interface ReaderService {
    List<String> readFromFile(String fromFileName);
}
```